versions.md

```diff
@@ -1,15 +1,25 @@
 Forthcoming
 -----------
 
-* Enable configuration of thread setting for all threads.
+Version 2.13.1
+--------------
+
+* Revert wrong fix for TCP deadlock on channel reuse
+* Fix wrong history selection when removing PDP samples
+* Fix data race when processing incoming PDP samples on different threads
+* Downgrade CMake minimum version required to 3.20
```

Comment:
We usually don't add entries for patch releases, as this file tracks new features introduced in each minor. Please remove